src/entities/applet/lib/types.ts

```diff
@@ -68,7 +68,7 @@ export type ItemResponses = {
 export type ActivityResponses = {
   id: string;
   name: string;
-  description: string | null;
+  description: string | { en: string } | null;
```

Comment:
Please come up with a different approach. The description should be provided language agnostic